cpp/src/arrow/acero/meson.build

```
@@ -133,7 +133,19 @@ foreach key, val : arrow_acero_benchmarks
     exc = executable(
         key,
         sources: val['sources'] + arrow_acero_testing_sources,
-        dependencies: [arrow_acero_dep, arrow_benchmark_dep, gmock_dep],
+        dependencies: [
+            arrow_acero_dep,
+            arrow_compute_test_dep,
+            arrow_benchmark_dep,
+            gmock_dep,
+        ],
     )
     benchmark(key, exc)
 endforeach
+
+pkg.generate(
+    filebase: 'arrow-acero',
+    name: 'Apache Arrow Acero Engine',
+    description: 'Apache Arrow\'s Acero Engine',
+    requires: ['arrow', 'arrow-compute'],
```

We don't need `arrow` here because `arrow-compute` has `Requires: arrow`:

```suggestion
    requires: ['arrow-compute'],
```